src/xutils.hpp

```diff
@@ -56,6 +56,7 @@ namespace xpyt
 
     void exec(const py::object& code, const py::object& scope = py::globals());
 
+    size_t get_hash_seed();
```

Comment:
std::size_t?

Reply:
Ah good catch. I'll fix it in the next PR.